components/GiveStarCard.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import Modal from 'react-bootstrap/Modal'
+import Button from 'react-bootstrap/Button'
+import InputGroup from 'react-bootstrap/InputGroup'
+import FormControl from 'react-bootstrap/FormControl'
+
+type Props = {
+  show: boolean
+}
+
+const GiveStarCard: React.FC<Props> = props => {
+  return (
+    <Modal
+      {...props}
+      size="lg"
+      aria-labelledby="contained-modal-title-vcenter"
+      centered
+    >
+      <Modal.Header closeButton>
+        <Modal.Title id="contained-modal-title-vcenter">
+          Who helped you the most?
+        </Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <InputGroup className="mb-3">
+          <InputGroup.Prepend>
+            <InputGroup.Text id="basic-addon1">@</InputGroup.Text>
+          </InputGroup.Prepend>
+          <FormControl
+            placeholder="Username"
+            aria-label="Username"
+            aria-describedby="basic-addon1"
+          />
+        </InputGroup>
+      </Modal.Body>
+      <Modal.Footer>
+        <Button onClick={() => {}}>Close</Button>
+      </Modal.Footer>
+    </Modal>
+  )
+}
+
+export default GiveStarCard
```

Comment:
Is it guaranteed that there will only be one variable exported from this file? If so, using `export default` is fine.
However, if you have plans to export more variables from this file, then you should export the normal way. This is to give more codebase consistency when someone imports variables from this file.

Reply:
I think it's either way it's fine. There's no real reason why we need `named exports`. We should use `default exports` unless we need more functionality such as exporting more than one component from a file.

Reply:
Ah, I see your point -- unless the file will export more than one variable, then both default or named export is fine. Thank you for the lesson Mr. Hyper Herman